warehouse/integrations/bigquery/bigquery.go

```diff
@@ -688,7 +716,9 @@ func (bq *BigQuery) LoadUserTables(ctx context.Context) (errorMap map[string]err
 
 	log.Infon("Loading data")
 	var partitionedUsersTable string
-	if bq.avoidPartitionDecorator() {
+
+	// avoid writing to partioned table if there is a no or custom partition instead of ingestion time partition i.e _PARTITIONTIME
+	if bq.avoidPartitionDecorator() || usersTablePartitionInWarehouse == nil || usersTablePartitionInWarehouse.Field != "" {
```

Comment:
Don't we need this check at other places as well which are using `bq.avoidPartitionDecorator()` ?

Reply:
I think we should. I was looking at users table but ideally partition could be changed for any table for that matter.